configurations/manager/200GB_backup_dataset.yaml

```diff
@@ -0,0 +1,12 @@
+
+# fill the cluster with data
+prepare_write_cmd: [
+    "cassandra-stress write cl=ALL n=209715200 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=650 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..209715200",
```

Comment:
`seq=1..209,715,200` is actually ~210GB on its own.
4 of the same range is ~850GB overall and not as the name of the file suggests.

Reply:
renamed to 850GB_backup_dataset.yaml